doc/source/notebooks/advanced/natural_gradients.pct.py

```diff
@@ -42,7 +42,7 @@
 M = 10
 
 x = np.random.uniform(size=(N, D))
-y = np.sin(10 * x)
+y = np.sin(10 * x[:, :1])
```

Comment:
I guess the natgrad notebook is focussing on just that, not multiple outputs, but perhaps it would be cleaner to somehow make use of the second input dimension?